test/conformance/kuttl/webhooks/namespaced-policy-clusterpolicy-different-resources/webhooks.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    webhook.kyverno.io/managed-by: kyverno
+  name: kyverno-resource-validating-webhook-cfg
+webhooks:
+- failurePolicy: Fail
+  rules:
+  - apiGroups:
+    - ''
+    apiVersions:
+    - 'v1'
+    operations:
+    - CREATE
+    - UPDATE
+    - DELETE
+    - CONNECT
+    resources:
+    - 'configmaps'
+    scope: '*'
+  - apiGroups:
+      - ''
+    apiVersions:
+      - 'v1'
+    operations:
+      - CREATE
+      - UPDATE
+      - DELETE
+      - CONNECT
+    resources:
+      - 'secrets'
+    scope: 'Namespaced'
```

Comment:
`configmaps` and `secrets` are namespaced resources, why do we need to take care of the scope in this case ?

Reply:
(i mean, there's no wildcard in those cases)

Reply:
I mean this would be a different implementation, wouldn't it? Currently based on the Policy type we are setting the scope. But, you want that we look up the resource type and based on that we are setting the scope, right?